src/terra-dev-site/doc/how-to.2/show-modal-content.doc.mdx

```diff
@@ -0,0 +1,57 @@
+# How To Show Modal Content
+
+The [ModalManager](/components/terra-application/reference/components/modal-manager) should be used to manage modal content presentation. 
+
+```jsx
+import React from 'react';
+import ApplicationBase from 'terra-application/lib/application-base';
+import ModalManager from 'terra-application/lib/modal-manager';
+
+import ComponentWithModal from './ComponentWithModal';
+
+const ExampleApp = () => (
+  <ApplicationBase>
+    <ModalManager
+      navigationPromptResolutionOptions={{
+        title: 'Example Title For Prompt',
```

Comment:
of these navigationPromptResolutionOptions, are any of these required?

Reply:
Not technically, no. I don't think we're able to establish hard requirements around this within terra-application at this point in time. I'd like to be able to at some point.